Description: put readlink in /bin/ like coreutils
 coreutils puts readlink in /bin, busybox puts it in /usr/bin.  udev rules,
 which must reference executables by absolute path, fail with busybox due
 to this incompatibility.  Move readlink to /bin for compatibility.
Author: Steve Langasek <steve.langasek@ubuntu.com>
Bug-Ubuntu: https://bugs.launchpad.net/bugs/1615021

Index: busybox-1.22.0/include/applets.src.h
===================================================================
--- busybox-1.22.0.orig/include/applets.src.h
+++ busybox-1.22.0/include/applets.src.h
@@ -280,7 +280,7 @@
 IF_RDATE(APPLET(rdate, BB_DIR_USR_SBIN, BB_SUID_DROP))
 IF_RDEV(APPLET(rdev, BB_DIR_USR_SBIN, BB_SUID_DROP))
 IF_READAHEAD(APPLET(readahead, BB_DIR_USR_SBIN, BB_SUID_DROP))
-IF_READLINK(APPLET(readlink, BB_DIR_USR_BIN, BB_SUID_DROP))
+IF_READLINK(APPLET(readlink, BB_DIR_BIN, BB_SUID_DROP))
 IF_READPROFILE(APPLET(readprofile, BB_DIR_USR_SBIN, BB_SUID_DROP))
 IF_REALPATH(APPLET(realpath, BB_DIR_USR_BIN, BB_SUID_DROP))
 IF_REFORMIME(APPLET(reformime, BB_DIR_BIN, BB_SUID_DROP))
